app/code/Magento/MediaGalleryUi/view/adminhtml/requirejs-config.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+var config = {
+    config: {
+        mixins: {
+            'mage/validation': {
+                'Magento_MediaGalleryUi/js/lib/mage/validation-mixin': true
```

Comment:
Can we avoid mixin creation and fix the issue either in `mage/validator` directory or adjusting validation usage on Media Gallery side